worldedit-core/src/main/java/com/fastasyncworldedit/core/extent/clipboard/io/schematic/MinecraftStructure.java

```diff
@@ -225,20 +225,20 @@ public void write(Clipboard clipboard) throws IOException {
         // Blocks
         LinListTag.Builder<@org.jetbrains.annotations.NotNull LinCompoundTag> blocks = LinListTag.builder(LinTagType.compoundTag());
```

Comment:
Change this top javax while we're here I guess?

Reply:
That would be great, but i hate that 
![image](https://github.com/user-attachments/assets/8607efbf-fdae-4d4f-b0a3-b2e10837881a)
![image](https://github.com/user-attachments/assets/82dd128a-b7d1-437d-8d5e-d065fcf3eacd)

guess i could suppress that simply